index.d.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference lib="dom"/>
 
+type QuerySelector = ParentNode['querySelector'];
```

Comment:
```suggestion
import { ParseSelector } from 'typed-query-selector';
```

Reply:
No, I'm not trying to use `typed-query-selector` directly. I need to use `querySelector`’s types.

`typed-query-selector` should not be a dependency of this project, but if the user installs it, it will be used automatically because it extends `querySelector`

Reply:
Oh, that may be hard.